test/bdd/features/issue-credential.feature

```diff
@@ -0,0 +1,17 @@
+
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@issue_credential
+Feature: Issue credential protocol
+  @begin_with_request
+  Scenario: Alice has a Carol's public invitation
+    Given   "Holder" exchange DIDs with "Issuer"
```

Comment:
Instead of generic Holder and Issuer,  we can give a specific example similar to https://github.com/hyperledger/aries-framework-go/blob/master/test/bdd/features/verifiable_credential.feature#L12 (Alice gets credential from University).

Reply:
Done!